x-pack/legacy/plugins/maps/public/selectors/map_selectors.js

```diff
@@ -48,16 +48,6 @@ function createLayerInstance(layerDescriptor, inspectorAdapters) {
   }
 }
 
-function createSourceInstance(sourceDescriptor, inspectorAdapters) {
```

Comment:
see other comment. This source-registry isn't strictly necessary. We'd just have a very long switch-case statement here to instantiate the right source.